vit-servicing-station-tests/src/common/clients/rest.rs

```diff
@@ -255,6 +269,10 @@ impl RestPathBuilder {
         self.path("health")
     }
 
+    pub fn service_version(&self) -> String {
```

Comment:
Notes: the exact place of entry point for versioning endpoint for client side app.